Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
```

Comment:
Add these lines to use tini
```
RUN apk add --update tini

ENTRYPOINT ["/sbin/tini", "--"]
```
Using this makes everything work more gracefully when shutting down.